I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/all_assets.rpy", line 26, in script
    define nora.ira = Fa
  File "game/all_assets.rpy", line 26, in <module>
    define nora.ira = Fa
NameError: name 'Fa' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/all_assets.rpy", line 26, in script
    define nora.ira = Fa
  File "C:\khp\renpy-8.1.0-sdk\renpy\ast.py", line 2508, in execute
    self.set()
  File "C:\khp\renpy-8.1.0-sdk\renpy\ast.py", line 2522, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\khp\renpy-8.1.0-sdk\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/all_assets.rpy", line 26, in <module>
    define nora.ira = Fa
NameError: name 'Fa' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.0.23051307
kursach 1.0
Fri Jun  2 03:24:45 2023
